Source/Core/BaseRow.swift

```diff
@@ -114,8 +114,8 @@ public class BaseRow : BaseRowType {
         return IndexPath(row: rowIndex, section: sectionIndex)
     }
     
-    private var hiddenCache = false
-    private var disabledCache = false {
+    var hiddenCache = false
```

Comment:
@Legoless Is this an automatic change by Xcode? we should use `fileprivate` (old private) or `private`.